substrate/pwasm-alloc/src/lib.rs

```diff
@@ -20,16 +20,16 @@ mod __impl {
 	extern crate alloc;
 	extern crate pwasm_libc;
 
-	use self::alloc::heap::{GlobalAlloc, Layout, Opaque};
+	use core::alloc::{GlobalAlloc, Layout};
```

Comment:
This line needs attention. Note, that I removed the leading `self`. Don't sure, how WASM and `no-std` stuff is linked, so it may be an issue.

Reply:
probably the `extern crate alloc` and corresponding feature is no longer necessary since this stuff is in `libcore` now

Reply:
+ the feature `global_allocator` is also no longer needed

Reply:
Yeah, forgot to push the second commit.